Tictactoe.cs

```diff
@@ -12,7 +12,7 @@ public class Tictactoe
         private MessageHandler MessageHandler { get; set; }
         public PlayerModel Player1 { get; set; }
```

Comment:
Private the players too perhaps! They shouldn't need to be accessed from outside TicTacToe